### ISSUE TITLE
Exception on BitmapText.clone()

### ISSUE THREAD

**Issue by FennelFetish** [jMonkeyEngine/jmonkeyengine#577]
Reproduce with following snippet:

```
BitmapFont font = assetManager.loadFont("Interface/Fonts/Default.fnt");
BitmapText text = new BitmapText(font);
BitmapText copy = text.clone(); // Exception thrown here
```

This worked with jme3-alpha1 and broke when I upgraded to beta1 (with the introduction of the Cloner?). The exception still happens in beta2 and the latest master branch.

The exception:
```
java.lang.RuntimeException: Error cloning object of type:class com.jme3.font.StringBlock
        at com.jme3.util.clone.Cloner.javaClone(Cloner.java:377)
        at com.jme3.util.clone.Cloner.clone(Cloner.java:260)
        at com.jme3.util.clone.Cloner.clone(Cloner.java:160)
        at com.jme3.font.BitmapText.cloneFields(BitmapText.java:98)
        at com.jme3.util.clone.Cloner.clone(Cloner.java:255)
        at com.jme3.util.clone.Cloner.clone(Cloner.java:160)
        at com.jme3.scene.Spatial.clone(Spatial.java:1360)
        at com.jme3.scene.Node.clone(Node.java:682)
        at com.jme3.scene.Node.clone(Node.java:62)
        at com.jme3.scene.Spatial.clone(Spatial.java:1448)
        at com.jme3.font.BitmapText.clone(BitmapText.java:79)
        at mygame.CloneTest.simpleInitApp(CloneTest.java:14)
        at com.jme3.app.SimpleApplication.initialize(SimpleApplication.java:220)
        at com.jme3.system.lwjgl.LwjglAbstractDisplay.initInThread(LwjglAbstractDisplay.java:130)
        at com.jme3.system.lwjgl.LwjglAbstractDisplay.run(LwjglAbstractDisplay.java:211)
        at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.IllegalAccessException: Class com.jme3.util.clone.Cloner can not access a member of class com.jme3.font.StringBlock with modifiers "public"
        at sun.reflect.Reflection.ensureMemberAccess(Reflection.java:102)
        at java.lang.reflect.AccessibleObject.slowCheckMemberAccess(AccessibleObject.java:296)
        at java.lang.reflect.AccessibleObject.checkAccess(AccessibleObject.java:288)
        at java.lang.reflect.Method.invoke(Method.java:491)
        at com.jme3.util.clone.Cloner.javaClone(Cloner.java:375)
        ... 15 more
```

**Comment by grizeldi** [comment]
I had this problem quite some time ago. See here:
https://hub.jmonkeyengine.org/t/cloning-bitmaptext-3-1/36824

**Comment by pspeed42** [comment]
I've fixed this on 3.1 and master.  I even added a test to the jme3-examples TestCloneSpatial app.

Note: even though BitmapText would clone before it was a broken clone.  So until now it's never really been properly cloneable.
